pkg/util/net/ip/ip.go

```diff
@@ -0,0 +1,81 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2019 Red Hat, Inc.
+ *
+ */
+package ip
+
+import (
+	"bufio"
+	"net"
+	"os"
+)
+
+const (
+	// sysfs
+	DISABLE_IPV6_PATH = "/proc/sys/net/ipv6/conf/default/disable_ipv6"
+
+	// Special IPs
+	IPV4_BIND_ADDRESS     = "0.0.0.0" //INADDR_ANY
+	IPV4_LOOPBACK_ADDRESS = "127.0.0.1"
+	IPV6_BIND_ADDRESS     = "[::]" //INADDR6_ANY
+	IPV6_LOOPBACK_ADDRESS = "[::1]"
+)
+
+func readSysfsFile(path string) (string, error) {
+	var output string = ""
+	file, err := os.Open(path)
+	if err != nil {
+		return output, err
+	}
+
+	defer file.Close()
+	scanner := bufio.NewScanner(file)
+	if scanner.Scan() {
+		output = scanner.Text()
+	}
+	if err := scanner.Err(); err != nil {
+		return output, err
+	}
+
+	return output, nil
```

Comment:
I think this should work:
```
data, err := ioutil.ReadAll(path)
if err != nil {
    return ...
}
return string(data), nil
```

Reply:
the snippet i use, is the one that other places in the code use,
do we want to change it ?

Reply:
So you are saying that we need to use this because it appears elsewhere in the code? I do not understand your point here.

Reply:
Lets look on 
https://github.com/kubevirt/kubevirt/blob/279617274c7c1fb6e16b9bb325b94bf41841acb9/pkg/virt-launcher/virtwrap/util/cpu_utils.go

IMO we shouldn't act in different logics, in different places,
This isn't the only place in the code that already use this logic,
Maybe there are reasons the developers chosen that way, maybe its just a matter of cosmetics and you are right, 
The most important thing is its would be in a one common way.

Reply:
That is not a valid reason, sorry. If you would have used a function that all are using, then it would have been a stronger argument, but this is not the case.

If someone did something wrong, it does not need to be duplicated in all the project.
I would suggest using a technical reasoning on why this approach is better than the simpler one.

Reply:
The reason is that the approach i suggest is preparation for a smoother refactor, that we can do in another PR imo, since its not belong to this scope.
The use of this code in this manner was also suggested by a maintainer.
The code at the moment doesn't have one function that is common in all the other places,
creating various delicates of the same functionality is a cookbook for problems imo.